aws/alb.go

```diff
@@ -204,11 +204,13 @@ func normalizeLoadBalancerName(clusterID string, certificateARN string) string {
 
 func createDefaultTargetGroup(alb elbv2iface.ELBV2API, name string, vpcID string, hc healthCheck) (string, error) {
 	params := &elbv2.CreateTargetGroupInput{
-		HealthCheckPath: aws.String(hc.path),
-		Port:            aws.Int64(int64(hc.port)),
-		Protocol:        aws.String(elbv2.ProtocolEnumHttp),
-		VpcId:           aws.String(vpcID),
-		Name:            aws.String(name),
+		HealthCheckPath:            aws.String(hc.path),
+		HealthCheckIntervalSeconds: aws.Int64(int64(10)),
```

Comment:
@hjacobs FYI: `int64()` is not needed here, it's only below where it is converting `hc.port` int -> int64 :)